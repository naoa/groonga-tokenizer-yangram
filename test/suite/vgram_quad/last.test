register tokenizers/yangram

table_create vgram_words TABLE_HASH_KEY ShortText
load --table vgram_words
[
{"_key": "は雨だ"},
{"_key": "は雨"}
]
tokenize TokenYaVgramQuad "今日は雨だな" NormalizerAuto --mode ADD
tokenize TokenYaVgramQuad "今日は雨だな" NormalizerAuto --mode GET
tokenize TokenYaVgramQuad "今日は雨だ" NormalizerAuto --mode ADD
tokenize TokenYaVgramQuad "今日は雨だ" NormalizerAuto --mode GET
tokenize TokenYaVgramQuad "今日は雨" NormalizerAuto --mode ADD
tokenize TokenYaVgramQuad "今日は雨" NormalizerAuto --mode GET
