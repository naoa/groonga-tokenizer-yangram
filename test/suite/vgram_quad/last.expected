register tokenizers/yangram
[[0,0.0,0.0],true]
table_create vgram_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table vgram_words
[
{"_key": "は雨だ"},
{"_key": "は雨"}
]
[[0,0.0,0.0],2]
tokenize TokenYaVgramQuad "今日は雨だな" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "日は雨",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "は雨だな",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "雨だ",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "だな",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "な",
      "position": 5,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
tokenize TokenYaVgramQuad "今日は雨だな" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "は雨だな",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
tokenize TokenYaVgramQuad "今日は雨だ" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "日は雨",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "は雨だ",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "雨だ",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "だ",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
tokenize TokenYaVgramQuad "今日は雨だ" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "は雨だ",
      "position": 2,
      "force_prefix": true,
      "force_prefix_search": true
    }
  ]
]
tokenize TokenYaVgramQuad "今日は雨" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "日は雨",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "は雨",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "雨",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
tokenize TokenYaVgramQuad "今日は雨" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "日は雨",
      "position": 1,
      "force_prefix": true,
      "force_prefix_search": true
    }
  ]
]
